system-variables.md

```diff
@@ -1109,11 +1109,11 @@ MPP is a distributed computing framework provided by the TiFlash engine, which a
 >
 > Currently, this feature is not fully compatible with [altering multiple columns or indexes in a single `ALTER TABLE` statement](/sql-statements/sql-statement-alter-table.md). When adding a unique index with the index acceleration, you need to avoid altering other columns or indexes in the same statement.
 >
-> Currently, this feature is not compatible with [PITR (Point-in-time recovery)](/br/backup-and-restore-overview.md). When using index acceleration, you need to ensure that there are no PITR log backup tasks running in the background. Otherwise, unexpected behaviors might occur, including:
+> Currently, the performance improvement is expected to increase by 10 times for the clusters of [PITR (Point-in-time recovery)](/br/backup-and-restore-overview.md) disabled, while there is no promotion for the clusters of both PITR and index acceleration enabled. We recommend that turn off the PITR, add index in a quick way, then turn on the PITR and do a fully backup. There are three expected behaviors, including:
```

Comment:
```suggestion
> Currently, the performance of clusters with [PITR (Point-in-time recovery)](/br/backup-and-restore-overview.md) disabled is expected to be about 10 times faster than that in v6.1.0. However, there is no performance improvement for clusters with both PITR and index acceleration enabled. To optimize performance, we recommend that turn off PITR, add indexes in a quick way, then turn on PITR and perform a full backup. Otherwise, there are three kinds of expected behaviors:
```